net-mail/b4/b4-0.10.1.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_SETUPTOOLS=rdepend
```

Comment:
You can drop this now -- maybe try PEP517: https://projects.gentoo.org/python/guide/distutils.html#the-pep-517-and-legacy-modes.

It's a nuisance but it's been foisted upon us by the Python ecosystem.

Reply:
I have no idea what that means, but it still builds with `DISTUTILS_USE_PEP517=setuptools`.  I've squashed that one in.